django_mailbox/models.py

```diff
@@ -647,14 +647,24 @@ def to_addresses(self):
         return addresses
 
     def reply(self, message):
-        """Sends a message as a reply to this message instance.
+        """Sends an EmailMessage as a reply to this message instance::
+
+            from django.core.mail import EmailMessage
+
+            message.reply(
+                EmailMessage(subject="pong", body="pongpong")
```

Comment:
I had to also add the to field so that the email was actually sent.

```python
from django.core import mail

def reply_to_message(message, subject, messagebody):
    rpl = mail.EmailMessage(
        subject=subject,
        body=messagebody,
        to=[message.from_header],
    )
    message.reply(rpl)
```

Reply:
Though it might be worth adding that to the `reply` method?

```
if not to:
  add to from reply_to or from_header
```

Reply:
> Interesting! Is it something to document or should we expect the "reply" function to set it automatically... WDYT @Pietro395 ?

Interesting observation @enaut 
In my opinion since you are using the reply function I would expect the “to” to be set automatically if not specified.

I think that's what a user would expect